common/scala/src/main/scala/whisk/core/entity/ActivationStore.scala

```diff
@@ -48,7 +49,8 @@ trait ActivationStore {
    * @param transid transaction ID for request
    * @return Future containing the retrieved WhiskActivation
    */
-  def get(activationId: ActivationId)(implicit transid: TransactionId): Future[WhiskActivation]
+  def get(activationId: ActivationId, user: Option[Identity] = None, request: Option[HttpRequest] = None)(
```

Comment:
Why make `user` and `request` an Option? When would you not pass these values?

Reply:
@markusthoemmes, since there exists situations were the `user` and `request` information may not be know, DB polling for blocking invocations, do you think we should leave those parameters as optional?

Reply:
@markusthoemmes, certain activation stores may need to maintain a separate DB to track activations for blocking invocations given the DB polling. When `user` or `request` is not provided to the activation store, this could signal that a blocking invocation is taking place.